app/src/main/java/com/example/poste/activities/DashboardActivity.java

```diff
@@ -178,7 +218,11 @@ private void showCreatePostDialog() {
         Spinner spinnerNewPostFolder = dialogView.findViewById(R.id.spinnerNewPostFolder);
 
         // Setup for the dropdown menu
-        ArrayAdapter<String> adapter= new ArrayAdapter<String>(this, android.R.layout.simple_spinner_item, new ArrayList<>(folderIdNameMap.keySet()));
+        ArrayAdapter<String> adapter = new ArrayAdapter<>(
+                this,
+                android.R.layout.simple_spinner_item,
+                new ArrayList<>(folderNameToIdMap.keySet())
```

Comment:
I was unable to get a dropdown when testing this. Could you please check again that the dropdown works when creating a post? And without a folder selected, cannot create a post either.

Reply:
<img width="1761" alt="image" src="https://github.com/Appyo-Poste/Poste/assets/32991956/e70dd7b9-ff26-4f0a-bdc1-fbabeefa2f73">
This is what I get in log when I try to get list of folders

Reply:
I hadn't implemented that yet, but it is implemented now.